e2e/test/scenarios/question/settings.cy.spec.js

```diff
@@ -75,7 +75,28 @@ describe("scenarios > question > settings", () => {
     it("should allow you to re-order columns even when one has been removed (metabase2#9287)", () => {
```

Comment:
Can we add the issue number to this spec in case anyone comes looking here in the future and is wondering why we are testing dragging in such a manual way?
```suggestion
    it("should allow you to re-order columns even when one has been removed (metabase #14238, #29287)", () => {
```

Reply:
Absolutely! I thought I had added a cy log for the new issue number. Whoops!